TweetBook/TweetBook.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Next time you see this in github desktop  you can just discard the changes since they seem to bring nothing new to the table.